### ISSUE TITLE
Investigate adding amp-consent to AMP pages

### ISSUE THREAD

**Issue by sareh** [bbc/simorgh#1297]
**Is your feature request related to a problem? Please describe.**
Currently, we do not have a GDPR-compliant banner for consenting to accept cookies. We should create a component and trigger it within an `<amp-consent>` tag.

**Describe the solution you'd like**
Create the front-end for the Cookie banner component.
Create a container for the GDPR-compliance logic for the banner. This should currently only contain AMP-specific logic. Later on, we will add the Canonical logic to this same container. 

For the AMP logic, we can use the same implementation as here: https://github.com/bbc/ws-alephamp-queue-renderer/blob/68f199f917bd4404f9b3587bf4751802ecf2cf3a/src/components/lib/templates/partials/consent.mustache 

**Describe alternatives you've considered**
N/A

**Testing notes**
We can add Cypress tests to check the logic of this banner.
- Does the banner appear the first time the user hits the page?
- Check banner behaviour around user acceptance/rejection of the cookie
- Do the buttons work as expected?

Other than Cypress, this will require testing across devices along with an accessibility run through to make sure that it's accessible for users using AT.

Dev insight: Locally, we should have unit tests that capture the visual representation for the cookie banner.  Once the PR is merged in and deployed up to Test, we can observe by visiting https://www.test.bbc.com/news/articles/c85pqyj5m2ko.amp that we see cookie banners in the UK since at the moment we are considered within the 'eu'. 

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

**Comment by sareh** [comment]
Closing issue as the investigation is on the PR #1306

**Comment by ChrisBAshton** [comment]
Can we please raise a new issue / issues for adding cookie banner? Suggested breakdown:

* Presentational cookie banner component 1 (privacy)
* Presentational cookie banner component 2 (cookies)
* AMP equivalent for 1.
* AMP equivalent for 2.
(NB it's possible we'll reuse the same component for both. I don't know)
* server logic for displaying the cookie banners, according to [feature file](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/features/cookie-banner.feature) (blocked on sign-off from legal - action for @jimjohnsonrollings).

**Comment by dr3** [comment]
@ChrisBAshton The first 4 can probably be 1 ticket tbh. I think were gunna use the same banner for all 4 cases, the only variance will be the type of button and the text.

The server logic 1 can be split into two though, amp and non amp imo :)
